app/helpers/functions.js

```diff
@@ -1,12 +1,15 @@
 var dateTime = require('node-datetime');
 var ip = require('ip');
-
+var sanitizer = require('sanitizer');
 
 const nodemailer = require('nodemailer');
 const fs = require('fs');
 const mustache = require('mustache');
 
-
+const crypto = require("crypto");
+const algorithm = 'aes-256-cbc';
+const key = "jkasdlkajsklj3242342^$@#%@#$%";

```

Comment:
Never pass the key like that in code directly. You should store any keys or any other credentials inside .env file.